### PR TITLE
fix: Improve console output and messages for colors

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: test/aspect.spec.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ describe("aspect", () => {
     expect(warnSpy).toHaveBeenCalledTimes(classes.length);
     expect(warnSpy.calls.flat()).toMatchInlineSnapshot(`
       [
-        "[REPLACED] aspect-w-2 -> use fractions instead, e.g. aspect-4/3",
-        "[REPLACED] aspect-h-4 -> use fractions instead, e.g. aspect-4/3",
+        "\x1b[93m[REPLACED]\x1b[0m aspect-w-2 -> use fractions instead, e.g. aspect-4/3",
+        "\x1b[93m[REPLACED]\x1b[0m aspect-h-4 -> use fractions instead, e.g. aspect-4/3",
```
Comment (AnnaRybkina) [inline test/aspect.spec.js:20]: `\x1b[` -part was not on the line for example https://github.com/warp-ds/preset-migrate/pull/11/files#diff-4a3ef178d4db5cec9d054131ad2656a9385f8756a11ec5c29d6e66b1504c7807R27. I guess we should align as I cannot see any differences in the output

Comment (AnnaRybkina) [reply test/aspect.spec.js:20]: Here: `\x1b[93m[REPLACED]\x1b[0m`
Some other places: `[93m[REPLACED][0m`

**Review by AnnaRybkina — APPROVED** [review]

File: utils.js [hunk 2]
```diff
@@ -1,11 +1,26 @@
-export const CSS_DOCS_URL = "https://warp-ds.github.io/css-docs"
-export const TECH_DOCS_URL = "https://warp-ds.github.io/tech-docs"
-export const COMING_SOON_MSG = "more info on replacements coming soon"
+export const CSS_DOCS_URL = "https://warp-ds.github.io/css-docs";
+export const TECH_DOCS_URL = "https://warp-ds.github.io/tech-docs";
+
+export const CONSOLE_COLORS = {
```
Comment (AnnaRybkina) [inline utils.js:4]: 🔥 Great improvement! It is so much readable now!

File: test/aspect.spec.js (hunk 1 above)
Comment (magnuh) [reply test/aspect.spec.js:20]: Apparently Vitest encodes and rewrites these somehow when it runs the tests... everything works fine though, but looks ugly. Not sure what to do about it. 🤷 😢

File: utils.js (hunk 2 above)
Comment (magnuh) [reply utils.js:4]: Agree 😄

**Pull request merged**